test_inductor test.sh fix

jataylo opened the pull request:
inductor/test_torchinductor suite is not running as part of the CI. I have triaged this down to a bug in the arguments supplied in test/run_test.py 

Currently test_inductor runs the test suites as:
`PYTORCH_TEST_WITH_INDUCTOR=0 python test/run_test.py --include inductor/test_torchinductor --include inductor/test_torchinductor_opinfo --verbose`

Which will only set off the test_torchinductor_opinfo suite

Example from CI logs: https://github.com/pytorch/pytorch/actions/runs/3926246136/jobs/6711985831#step:10:45089
```
+ PYTORCH_TEST_WITH_INDUCTOR=0
+ python test/run_test.py --include inductor/test_torchinductor --include inductor/test_torchinductor_opinfo --verbose
Ignoring disabled issues:  []
/var/lib/jenkins/workspace/test/run_test.py:1193: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if torch.version.cuda is not None and LooseVersion(torch.version.cuda) >= "11.6":
Selected tests:
 inductor/test_torchinductor_opinfo
Prioritized test from test file changes.
reordering tests for PR:
prioritized: []
the rest: ['inductor/test_torchinductor_opinfo']
```


cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

jithunnair-amd commented:
@malfet @kit1980 Looks like this PR exposed a UT failure in test_torchinductor on CUDA. Maybe that should be addressed/skipped first in a separate PR?
```
2023-01-24T12:18:11.1486741Z ======================================================================2023-01-24T12:18:11.1486881Z ERROR [0.102s]: test_roi_align_dynamic_shapes_cuda (__main__.CudaTests)2023-01-24T12:18:11.1487078Z ----------------------------------------------------------------------...2023-01-24T12:18:11.1505193Z FAILED (errors=1, skipped=235, expected failures=3)...2023-01-24T12:18:11.1508829Z inductor/test_torchinductor failed!
```

jataylo committed:
enable test_torchinductor UT suite

inductor/test_torchinductor suite is not running as part of the CI. I have triaged this down to a bug in the arguments supplied in test/run_test.py

Currently test_inductor runs the test suites with:

Which will only set off the test_torchinductor_opinfo suite
Example from CI logs: https://github.com/pytorch/pytorch/actions/runs/3926246136/jobs/6711985831#step:10:45089

jataylo commented:
Rebased with upstream after `.jenkins` change to `.ci`

jataylo commented:
No UT fails with this change after rebase and both UT suites are now run on the CI.
https://github.com/pytorch/pytorch/actions/runs/4116033510/jobs/7106025838

**After change**
```
2023-02-07T17:39:25.1119386Z + PYTORCH_TEST_WITH_INDUCTOR=0
2023-02-07T17:39:25.1119862Z + python test/run_test.py --include inductor/test_torchinductor inductor/test_torchinductor_opinfo --verbose
2023-02-07T17:39:27.0608717Z Ignoring disabled issues:  []
2023-02-07T17:39:27.1563790Z /var/lib/jenkins/workspace/test/run_test.py:1195: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
2023-02-07T17:39:27.1564217Z   if torch.version.cuda is not None and LooseVersion(torch.version.cuda) >= "11.6":
2023-02-07T17:39:27.1564487Z Selected tests:
2023-02-07T17:39:27.1564684Z  inductor/test_torchinductor
2023-02-07T17:39:27.1564915Z  inductor/test_torchinductor_opinfo
```

**Before change**
```
+ PYTORCH_TEST_WITH_INDUCTOR=0
+ python test/run_test.py --include inductor/test_torchinductor --include inductor/test_torchinductor_opinfo --verbose
Ignoring disabled issues:  []
/var/lib/jenkins/workspace/test/run_test.py:1193: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if torch.version.cuda is not None and LooseVersion(torch.version.cuda) >= "11.6":
Selected tests:
 inductor/test_torchinductor_opinfo
Prioritized test from test file changes.
reordering tests for PR:
prioritized: []
the rest: ['inductor/test_torchinductor_opinfo']
```
@malfet @kit1980 could you take a look to see if this can be merged to enable `torch_testinductor.py` UTs.

seemethere commented:
@pytorchbot merge
